power-pay-frontend/package.json

```diff
@@ -1,13 +1,15 @@
 {
   "name": "power-pay-frontend",
   "private": true,
-  "homepage": "https://ADORSYS-GIS.github.io/e2e-banking-app",
+  "homepage": "https://adorsys-gis.github.io/e2e-banking-app",
   "version": "0.0.0",
   "type": "module",
   "scripts": {
     "predeploy": "npm run build",
     "deploy": "gh-pages -d build",
     "build-for-gh": "vite build --base=/e2e-banking-app",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "react-scripts start",
```

Comment:
Why react-scripts, while we're using `vite` ? Read again about react with vite [here](https://vitejs.dev/guide/).